### DISCUSSION TITLE
After adding new data, the problems displayed after testing the model

### DISCUSSION THREAD

**Discussion by hanrm94** [deepmodeling/deepmd-kit#4035] [Q&A]
After adding new data and conducting model testing, some points have particularly large errors. What are the problems with the newly added dataset or other reasons？
before adding new data：
![dptest_01](https://github.com/user-attachments/assets/ed558f96-c557-4f3c-b9d8-dc8c6414e26b)
after adding new data：
![dptest_03](https://github.com/user-attachments/assets/031cefc8-577e-4fd7-92c9-c7a3a2d18802)

**Comment by njzjz** [comment]
The new data has very different energies and large forces, which may decrease the RMSE.
